earthly/rust/scripts/std_build.sh

```diff
@@ -18,6 +18,8 @@ source "$(dirname "$0")/colors.sh"
 
 rc=0
 
+export RUSTDOCFLAGS="--enable-index-page -Zunstable-options" 
```

Comment:
Shouldn't this be defined in `stdcfgs/cargo_config.toml` and then in the projects `config.toml` to match?

Reply:
fixed

Reply:
I have figured out that, added these options into the `cargo_config.toml` file. I was the problem with the `cargo-modules` version, the latest one does not work